test/SIL/Serialization/literals.sil

```diff
@@ -14,6 +14,8 @@ bb0:
   %2 = const_string_literal utf8 "\u{0B}"
 // CHECK: const_string_literal utf16 "\u{0B}"
   %3 = const_string_literal utf16 "\u{0B}"
+// CHECK: string_literal bytes "ABCD"
```

Comment:
Can you add a specific test to SIL parser as well? We generally put one in both places. I think it is b/c the other parser one does a round trip through the parser, i.e. parse/print/parse instead of parse/serialize/print. I might be wrong though. If I am wrong don't worry about it, but I believe I am right.

Reply:
Done, thanks for the context!